src/core/Akka/Event/LoggingBus.cs

```diff
@@ -165,25 +166,27 @@ private void AddLogger(ActorSystemImpl system, Type loggerType, LogLevel logLeve
         {
             var loggerName = CreateLoggerName(loggerType);
             var logger = system.SystemActorOf(Props.Create(loggerType).WithDispatcher(system.Settings.LoggersDispatcher), loggerName);
-            var askTask = logger.Ask(new InitializeLogger(this));
+            var askTask = logger.Ask(new InitializeLogger(this), timeout);
```

Comment:
If the logging system failed to start, the task we issued would never be de-allocated. This code has been corrected to align with the JVM.